core/src/main/java/org/mule/runtime/core/internal/processor/simple/ParseTemplateProcessor.java

```diff
@@ -41,8 +56,14 @@ private void loadContentFromLocation() throws InitialisationException {
       if (location == null) {
         throw new IllegalArgumentException("Location cannot be null");
       }
-      content = IOUtils.getResourceAsString(location, this.getClass());
-
+      content = getResourceAsString(location, this.getClass());
```

Comment:
You should use the configured encoding to read the file because the file will be already encoded and the given encoding will be useless.